github/commands/commit.py

```diff
@@ -14,7 +14,7 @@
 from ...common import util
 
 
-class GsShowGithubIssuesCommand(TextCommand, GitCommand, git_mixins.GithubRemotesMixin):
+class GsGithubShowGithubIssuesCommand(TextCommand, GitCommand, git_mixins.GithubRemotesMixin):
```

Comment:
Should there classes have Github twice in it's name

Reply:
Good point, I think I caught all of these in the command names, this one may have slipped through.